### PR TITLE
Support standalone yaml

### PR DESCRIPTION
**Pull request by lhoestq**
see (internal) https://huggingface.slack.com/archives/C02V51Q3800/p1703885853581679

### CODE REVIEW

File: src/datasets/load.py [hunk 1]
```diff
@@ -1004,7 +1005,15 @@ def __init__(
 
     def get_module(self) -> DatasetModule:
         readme_path = os.path.join(self.path, "README.md")
+        standalone_yaml_path = os.path.join(self.path, ".huggingface.yml")
```
Comment (julien-c) [inline src/datasets/load.py:1008]: i think it's .yaml on the hub...

Comment (lhoestq) [reply src/datasets/load.py:1008]: Good catch, I got confused by an internal PR name

Comment (julien-c) [reply src/datasets/load.py]: never trust an internal PR 🤣

**Comment by d710055071** [comment]
@lhoestq 
hello
I think it should be defined in config.py
DATASET_ README_ FILENAME="README. md"
This can replace all "README. md"

**Comment by lhoestq** [comment]
Thanks for the feedback :) merging now

**Pull request merged**